### ISSUE TITLE
inconsistent 4-letter coding between ML and PC data

### ISSUE THREAD

**Issue by mkclapp** [calacademy-research/COMB#96]
I tried setting species = "AUWA" in the latest version of our combined model ('top1_nop10_scores_nb_covars_ppc.R'; though the exact model shouldn't matter as it's a readCombined issue) but only PC data were properly read in; the ML matrix was populated with 0's only. This is because this species is listed as YRWA in the ML data. I confirmed that setting species = "YRWA" yields a data list with the inverse issue-- ML data but no PC data.

Not sure what other consistencies there could be or where would be the best step to edit the species names to be consistent.

**Comment by ddkapan** [comment]
Good find.

We should apply a fix to the PC dataset since we are not using accepted names and the subspecific information isn't relevant for this work. 

AUWA = YRWA
RSFL = NOFL

&& probably others thanks to MP!

It should go into delintPC.R since it modifies the original readPC.R input.

@mkclapp can you make the change?

Cheers,

Durrell

**Comment by mkclapp** [comment]
I have created a pull request from branch "96-inconsistent-4-letter-coding" that modifies delintPC.R to include a step that checks all 4-letter codes against the same .txt file that readML.R uses. 

**NOTE** The code I pushed contains a Google Drive link to automatically download the file if it doesn't already exist on your local workspace (copied directly from readML.R). If you haven't had any problems syncing with Drive, it should work seamlessly. If not (or if you prefer not to sync with Drive), you'll need to download "birds.txt" from Google Drive [here](https://drive.google.com/drive/u/1/folders/10TRgKc4NemrDeYMnBOtFmf-PS6HI5F6F) and add it to COMB/acoustic/data_ingest/input. 

If folks would prefer I remove the Drive link entirely, just let me know and I can modify and pull-request again.
